atomwrappers.py

```diff
@@ -59,15 +59,75 @@ def recursive_print(self, result='', indent=''):
 class CEvaluationLink(CLink):
     atom_type = 'EvaluationLink'
 
+class CExecutionLink(CLink):
+    atom_type = 'ExecutionLink'
+
+class CLazyExecutionOutputLink(CLink):
+    atom_type = 'LazyExecutionOutputLink'
+
+class CQuantitativePredicateLink(CLink):
+    atom_type = 'QuantitativePredicateLink'
+
 class CPredicateNode(CNode):
     atom_type = 'PredicateNode'
 
+class CSchemaNode(CNode):
+    atom_type = 'SchemaNode'
+
+class CQuantitativeSchemaNode(CNode):
+    atom_type = 'QuantitativeSchemaNode'
+
+class CQuantitativePredicateNode(CNode):
+    atom_type = 'QuantitativePredicateNode'
+
 class CConceptNode(CNode):
     atom_type = 'ConceptNode'
 
+class CNumberNode(CNode):
+    atom_type = 'NumberNode'
+
 class CMoleculeNode(CNode):
     atom_type = 'MoleculeNode'
 
+class CChebiNode(CNode):
+    atom_type = 'ChebiNode'
+
+class CUniprotNode(CNode):
+    atom_type = 'UniprotNode'
+
+class CRnaNode(CNode):
+    atom_type = 'RnaNode'
+
+class CPubchemNode(CNode):
+    atom_type = 'PubchemNode'
+
+class CCodingRna(CNode):
+    atom_type = 'CodingRna'
+
+class CNonCodingRna(CNode):
+    atom_type = 'NonCodingRna'
+
+class CPathwayNode(CNode):
+    atom_type = 'PathwayNode'
+
+class CReactomePathway(CNode):
+    atom_type = 'ReactomePathway'
+
+class CSmpPathway(CNode):
+    atom_type = 'SmpPathway'
+
+class CGoNode(CNode):
+    atom_type = 'GoNode'
+
+class CBiologicalProcess(CNode):
```

Comment:
You modified the class name for GO subcategories, but you didn't modify the scripts which uses the previous class name. Revert the previous definition or modify the other scripts as well.